build/squire-raw.js

```diff
@@ -2230,19 +2231,20 @@ var onPaste = function ( event ) {
             this.fireEvent( 'willPaste', {} );
             return;
         }
+        
+        var imagePasteEvent = {
```

Comment:
There is a slight contract modification to this.  Now instead of a new event per event, you are reusing the same one.  I am not sure if that matters, but its possible someone may modify the event args in their handler, which would be problematic for other consumers.